core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -65,12 +71,21 @@ export class ExplorationHtmlFormatterService {
     element = (
       this.extensionTagAssembler.formatCustomizationArgAttrs(
         element, interactionCustomizationArgs));
-    element.attr(
-      'last-answer', parentHasLastAnswerProperty ? 'lastAnswer' : 'null');
     if (labelForFocusTarget) {
       element.attr('label-for-focus-target', labelForFocusTarget);
     }
-    return element.get(0).outerHTML;
+    element.attr(
+      'last-answer', parentHasLastAnswerProperty ? 'lastAnswer' : 'null');
+    let val = element.get(0).outerHTML;
+    if (this.migratedInteractions.indexOf('GraphInput') >= 0) {
```

Comment:
Why is special casing applied only for GraphInput? Can there be other interactions as well? If so, let's use a list.

Reply:
Oh no... this is a logical bug in my code. The condition I wrote will always be true. Ideally, this has to be: `if (this.migratedInteractions.indexOf(interactionId) >= 0) {` Thanks for catching it. But how are the unit tests passing?

Reply:
I get it now... I have hardcoded graphInput in the replacements as well.

Reply:
Changed!